app/controllers/clusters_controller.rb

```diff
@@ -82,10 +82,16 @@ class ClustersController < AuthenticatedUsersController
   post '/clusters/:cluster_id/unmanage' do
     Tendrl.load_node_definitions
     flow = Tendrl::Flow.new('namespace.tendrl', 'UnmanageCluster')
+    body = JSON.parse(request.body.read)
```

Comment:
This fails if empty JSON object is not provided in the request body.
How about:

```ruby
body = JSON.parse(request.body.read.present? ? request.body.read : '{}')
```

Either this, or else change the failing test to provide an empty JSON while making the call.

Reply:
Ack